src/views/users/const.js

```diff
@@ -100,3 +100,13 @@ export const userJSONSelectMeta = (vm, withoutOrgRole = false) => {
     }
   }
 }
+export const MFALevel = {
+  allUsers: 3,
+  onlyAdminUsers: 4
+}
+
+export const MFASystemSetting = {
+  notEnable: 0,
+  allUsers: 1,
+  onlyAdminUsers: 2
+}
```

Comment:
The code contains a typo where "VM" should be replaced with 'User', hence you can remove ' VM ' and replace 'export' with appropriate usage. Here is improved version:

```
@@ -103,6 +107,8 @@ export const userJSONSelectMeta = (vm, withoutOrgRole = false) => {
   }
 }}
```